uvl/Java/UVLJavaLexer.g4

```diff
@@ -20,32 +20,22 @@ package uvl;
 
   @Override
```

Comment:
I guess most of these comments were rather self-explanatory and I can live with their removal. Is there a specific reason we do not want any comments here?

Reply:
Ah ok, probably makes sense to restore them now. How do we proceed here? I guess it would make sense to re-add them within this PR then.